src/components/inline-alert/index.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames'
+import { IconInfo24 } from '@hashicorp/flight-icons/svg-react/info-24'
+import { InlineAlertProps } from './types'
+import s from './inline-alert.module.css'
+
+export default function InlineAlert({
+	color = 'neutral',
+	description,
+	icon,
+	title,
+}: InlineAlertProps) {
+	return (
+		<div className={classNames(s.default, s[color])}>
+			<span className={s.icon} data-testid="icon">
+				{icon ?? <IconInfo24 />}
```

Comment:
🎨 This could be simplified to `{icon}` by defaulting the descrtructured prop to `<IconInfo24 />` above.